content/support-matrix/delfin/dell-emc/vmax/_index.md

```diff
@@ -0,0 +1,370 @@
+---
+title: VMAX
+menuTitle: VMAX 
+description: "VMAX Support Matrix"
+weight: 19
+disableToc: false
+tags: ["support matrix", "delfin", "drivers", "dell emc"]
+---
+
+### Model and Management software versions supported
```

Comment:
**Question**: Are we going to update this page with actual data? Except for the Model and Management software versions supported table the other tables all seem to be the same as the template. Are we leaving these to be filled by community members or driver authors?

Reply:
@Shruthi-1MN @NajmudheenCT 
The individual driver support matrix files that are part of the PR are currently empty. How are we going ahead with this?

Reply:
Yes anvit. We can remove that. @Shruthi-1MN Please remove other indexes , keep only ( Add new driver, VMAX and Oceanstoe , sections)